content/en/blog/_posts/2019-05-22-OPA-Gatekeeper-Policy-and-Governance-for-Kubernetes.md

```diff
@@ -0,0 +1,194 @@
+
+---
+layout: blog
+title: "OPA Gatekeeper: Policy and Governance for Kubernetes"
+date: 2019-05-22
```

Comment:
```suggestion
date: 2019-08-06
```
As it's been quite awhile since this was submitted. It'd probably be best just to quickly update the date both here and in the file name. 👍